packages/components/src/Workflow/subcomponents/Footer/components/ProgressStepper/ProgressStepper.tsx

```diff
@@ -17,21 +16,24 @@ const getStepStatus = (
   isComplete: boolean,
   isCurrentStep: boolean,
   step: string
-): { icon: React.SVGAttributes<SVGSymbolElement>; accessibleName: string } => {
+): {
+  icon: React.SVGAttributes<SVGSymbolElement>
```

Comment:
Won't solve your render issue but should be a `JSX.Element`...?

Reply:
Oooo yep I should update that type